src/d5a50_len_5fd0.c

```diff
@@ -114,7 +126,13 @@ INCLUDE_ASM(s32, "d5a50_len_5fd0", set_hud_element_size);
 
 INCLUDE_ASM(s32, "d5a50_len_5fd0", func_80144E4C);
 
-INCLUDE_ASM(s32, "d5a50_len_5fd0", func_80144E74);
+void func_80144E74(s32 iconIndex, s32 arg1) {
+    HudElement *hudElement;
```

Comment:
Pointer should be next to type name (HudElement*) and also you can initialize hudElement on the same line: `HudElement* hudElement = gHudElementList[iconIndex & ~0x800];`